src/utils/swapMinAmounts.ts

```diff
@@ -1,18 +1,37 @@
-import { asNumber, asString } from 'cleaners'
+import { asNumber, asObject, asString } from 'cleaners'
 import {
   EdgeAccount,
   EdgeCurrencyWallet,
   EdgePluginMap,
   EdgeSwapRequest,
   EdgeSwapRequestOptions
 } from 'edge-core-js'
+import fetch from 'node-fetch'
+
+import { config } from './config'
 
 interface SwapReqInfo {
   fromWallet: EdgeCurrencyWallet
   toWallet: EdgeCurrencyWallet
   amount: number
 }
 
+interface SingleCurrencyPairMinAmt {
+  [pluginName: string]: number
+}
+
+const baseUri: string = config.ratesServerAddress
+const route: string = 'v1/exchangeRate/'
+const queryStr: string = '?currency_pair=USD_'
+
+const usdAmount: number = 200 // Variable to help get initial binary search max in USD
```

Comment:
You probably don't need to type this since it's a defined constant.  Also, probably use all caps since it's a constant.  Could have a more descriptive name than usdAmount, maybe something like INITIAL_USD_AMOUNT.

Reply:
Changed variable to `INIT_MAX_USD_AMOUNT` and removed the typing.